system-int-test/test/cmr/system_int_test/bootstrap/bulk_index/concepts_test.clj

```diff
@@ -37,6 +37,30 @@
         exp-items (set (map #(dissoc % :status) expected))]
     (is (= exp-items search-items))))
 
+(defn- create-read-update-token
+  "Create a token with read/pdate permission."
+  []
+  (let [admin-read-update-group-concept-id (e/get-or-create-group (s/context) "admin-read-update-group")]
+    (e/grant-group-admin (s/context) admin-read-update-group-concept-id :read :update)
+    ;; Create and return token
+    (e/login (s/context) "admin-read-update" [admin-read-update-group-concept-id])))
+
+(deftest index-system-concepts-test-invalid
+  (s/only-with-real-database
+   ;; Disable message publishing so items are not indexed as part of the initial save.
+   (core/disable-automatic-indexing)
+
+   ;; Remove fixture ACLs
+   (let [response (ac/search-for-acls (u/conn-context) {} {:token (tc/echo-system-token)})
+         items (:items response)]
+     (doseq [acl items]
+       (e/ungrant (s/context) (:concept_id acl))))
+
+   (let [{:keys [status errors]} 
+          (bootstrap/bulk-index-system-concepts {tc/token-header create-read-update-token})]
+     (is (= [401 ["You do not have permission to perform that action."]]
+            [status errors])))))
```

Comment:
I don't understand why this doesn't give me the update permission. I've looked at a few other places where tokens are used, it looks like in the end it's doing just this...   @daniel-zamora, can you see what I'm doing wrong? Thanks.

Reply:
I'm not sure if caching might be involved. In any case I think you can remove this test since we verify the system management ACL permission works as expected several other places in the tests.

Reply:
Removed.